api-server.py

```diff
@@ -101,9 +118,37 @@ def main():
     elif args.coznet:
         settings.setup_coznet()
 
+    syslog_facility = None
+    if args.syslog or args.syslog_local:
+        # Setup the syslog facility
+        if args.syslog_local:
+            print("Logging to syslog local facility %s" % args.syslog_local)
+            syslog_facility = SysLogHandler.LOG_LOCAL0 + args.syslog_local
+        else:
+            print("Logging to syslog user facility")
+            syslog_facility = SysLogHandler.LOG_USER
+
+        # Setup logzero to only use the syslog handler
+        logzero.logfile(None, disableStderrLogger=args.disable_stderr)
+        syslog_handler = SysLogHandler(facility=syslog_facility)
+        logger.addHandler(syslog_handler)
+    else:
+        # Setup file logging
+        if args.logfile:
+            logfile = os.path.abspath(args.logfile)
+            print("Logging to logfile and stdout: %s" % logfile)
```

Comment:
This message isn't technically accurate for two reasons:

1) It's stderr, not stdout.

2) You can disable stderr, in which case only the logfile is used.

Reply:
👍